src/static/css/pages/sub-pages.less

```diff
@@ -16,6 +16,10 @@
     }
 }
 
+.sub-page__grandchild, .sub-page__grandchild + .content_sidebar {
```

Comment:
I don't think we ever addressed breaking lines for each selector in the CSS playbook group, but it's something I've always followed to make readability a bit better.